libraries/cmake/source/lief/CMakeLists.txt

```diff
@@ -0,0 +1,388 @@
+# Copyright (c) 2014-present, The osquery authors
+#
+# This source code is licensed as defined by the LICENSE file found in the
+# root directory of this source tree.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+include(ExternalProject)
+include(CheckCCompilerFlag)
+include(CheckCXXCompilerFlag)
```

Comment:
These should be removed (see the comments below).

Reply:
removed